src/Apps/ViewingRoom/Routes/Statement/ViewingRoomStatementRoute.tsx

```diff
@@ -45,14 +29,16 @@ const StatementRoute: React.FC<ViewingRoomStatementRouteProps> = ({
   )
 }
 
-// Top-level route needs to be exported for bundle splitting in the router
-export default createFragmentContainer(StatementRoute, {
-  viewingRoom: graphql`
-    fragment ViewingRoomStatementRoute_viewingRoom on ViewingRoom {
-      ...ViewingRoomIntro_viewingRoom
-      ...ViewingRoomWorks_viewingRoom
-      ...ViewingRoomPullQuote_viewingRoom
-      ...ViewingRoomSubsections_viewingRoom
-    }
-  `,
-})
+export const ViewingRoomStatementRouteFragmentContainer = createFragmentContainer(
```

Comment:
Removed bundle splitting from the sub routes. Sometimes there's diminishing returns here, in the sense that a tiny snippet of page is probably just a few kb -- however, a fetch needs to happen, a mount needs to happen, etc. A lot is going on and can lead so some layout thrashing, especially if the MP request is so fast that it outpaces the bundle request happening in parallel, leading to a white flash. 

So bundle splitting: required at the top most route (e.g., `/viewing-room`) for the new app shell architecture, and then evaluated on a case-by-case basis for `children` routes. In many cases it's just not needed and can diminish the UX.